runners/gradle-plugin/build.gradle.kts

```diff
@@ -83,6 +80,13 @@ pluginBundle {
     }
 }
 
+val javadocJar by tasks.registering(Jar::class) {
+    group = JavaBasePlugin.DOCUMENTATION_GROUP
+    description = "Assembles Javadoc into a JAR"
+    archiveClassifier.set("javadoc")
+    from(tasks.javadoc)
+}
```

Comment:
I believe this will now publish Java's Javadoc format, whereas before it published Dokka's HTML format. I've checked the `javadoc.jar` for the [latest release](https://search.maven.org/artifact/org.jetbrains.dokka/dokka-gradle-plugin/1.7.20/jar)

The previous `javadocJar` task with Dokka's HTML format is declared in `kotlin-jvm.gradle.kts`, but if you extract it to `maven-publish` as I mentioned, I think it should be accessible here

Reply:
Ahh okay, now I understand your suggestion to move the javadocJar task to the `maven-publish` convention. It needs to be present in both `kotlin-dsl` projects, and `kotlin-jvm` projects. But the Gradle plugin must only have the `kotlin-dsl` plugin applied, not both.

Reply:
We could extract Dokka into its own convention, would that help? And then use the convention from `maven-publish`?

Reply:
Good ideas, done. Take a look at let me know if it's as you'd like!